home-data/src/main/java/com/softserveinc/ita/homeproject/homedata/entity/RoleEnum.java

```diff
@@ -15,4 +17,9 @@
     public String getName() {
         return name;
     }
+
+    public static RoleEnum getEnum(String value) {
+        return Arrays.stream(RoleEnum.values()).filter(m -> m.name.equals(value)).findAny().orElse(null);
```

Comment:
Can we cache it in a map and just take it by key instead of streaming it each we access it?

Reply:
Okay we can

Reply:
I need to create a map in the constructor, put the values ​​there, and then get it in the method by the key?

Reply:
I remember that we've decided on it in one of our meetings.